crates/sui-core/src/unit_tests/move_integration_tests.rs

```diff
@@ -8,7 +8,7 @@ use crate::authority::authority_tests::{
     TestCallArg,
 };
 
-use move_core_types::language_storage::TypeTag;
+use move_core_types::{language_storage::TypeTag, value::MoveValue};
```

Comment:
I'm surprised we have to keep adding to this file. I was trying to kill it :)
What is blocking these from being sui-adapter-transactional-tests ?

Reply:
Didn't know (and could not find examples of) how to instantiate struct args (and vector of structs args)...